src/kudu/consensus/consensus_queue.cc

```diff
@@ -175,6 +175,7 @@ PeerMessageQueue::Metrics::Metrics(const scoped_refptr<MetricEntity>& metric_ent
 PeerMessageQueue::PeerMessageQueue(const scoped_refptr<MetricEntity>& metric_entity,
                                    scoped_refptr<log::Log> log,
                                    scoped_refptr<ITimeManager> time_manager,
+                                   const scoped_refptr<PersistentVarsManager>& persistent_vars_manager,
```

Comment:
why are we funneling persistent vars manager into consensus queue, instead of funneling persistent vars_?
That way we don't have to LoadPersistentVars again.

Reply:
We don't load the persistent vars (from protobuf) here either. persistent_vars_manager holds a cached map of all loaded persistent vars. It's loaded once and subsequent calls just returns a pointer from the map.

It's essentially acting like a per tablet singleton. Ofc we only have one tablet so how we pass it is a cosmetic difference. I passed the manager mainly to keep things consistent with other places that uses persistent vars.